CONTRIBUTING.md

```diff
@@ -81,39 +81,58 @@ cherry-picked into `book-v2` branch.
 
 Check the CI job after to do the Pull Request and then, click on in the `Details` of `netlify/kubebuilder/deploy-preview`
 
-## Understanding the versions
+## Versioning
 
 |   Name	|   Example	|  Description |
 |---	|---	|---	|
-|  PROJECT version |  `v1`,`v2`,`v3` | As of the introduction of [Extensible CLI and Scaffolding Plugins](https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/extensible-cli-and-scaffolding-plugins-phase-1.md), PROJECT version represents the layout of PROJECT file itself.  For `v1` and `v2` projects, there's extra meaning -- see below.  |
-|  Release version | `v2.2.0`, `v2.3.0`, `v2.3.1` | Tagged versions of the KubeBuilder project, representing changes to the source code in this repository. See the [releases](https://github.com/kubernetes-sigs/kubebuilder/releases) page. |
-|  Plugin Versions | `go.kubebuilder.io/v2.0.0` | Represents the version of an individual plugin, as well as the corresponding scaffolding that it generates. |
+|  KubeBuilder version | `v2.2.0`, `v2.3.0`, `v2.3.1` | Tagged versions of the KubeBuilder project, representing changes to the source code in this repository. See the [releases] page for binary releases. |
+|  Project version |  `"1"`, `"2"`,`"3-alpha"` | Project version defines the scheme of a `PROJECT` configuration file. This version is defined in a `PROJECT` file's `version`. |
+|  Plugin version | `v2.0`, `v2.1` | Represents the version of an individual plugin, as well as the corresponding scaffolding that it generates. This version is defined in a plugin key, ex. `go.kubebuilder.io/v2.0`. See the [design doc][plugins-design-versioning] for more details. |
```

Comment:
Currently, we have implemented v2.0.0. So, why change for v2.0?
Otherwise, we are unable to bum v2.0.1, for example, which represents a change that is NOT a breaking change and is just a bugfix. Was not it that you indeed suggested in the latest KB meeting?

Reply:
See [this conversation](https://github.com/kubernetes-sigs/kubebuilder/issues/1519#issuecomment-632334210), which discusses dropping minor version from plugin versions.

Reply:
OK. I am ok with we move in this way.